### ISSUE TITLE
bazel error:cannot be loaded: Not found: Op type not registered 'ClipByValue' in binary running on

### ISSUE THREAD

**Issue by janson91** [tensorflow/tensorflow#19822]
when i start a tensorflow serving :bazel-bin/tensorflow_serving/model_servers/tensorflow_model_server --port=8000 --model_name=aS_24443.1000 --model_base_path=...
i get the error :

2018-06-07 07:49:54.236996: I tensorflow_serving/util/retrier.cc:34] Retrying of Loading servable: {name: aS_24443.1000 version: 2} retry: 5
2018-06-07 07:49:54.283112: I external/org_tensorflow/tensorflow/contrib/session_bundle/bundle_shim.cc:360] Attempting to load native SavedModelBundle in bundle-shim from: /home/work/ics-contrib/freechat-serving/serving/load/aS_24443.1000/2
2018-06-07 07:49:54.283150: I external/org_tensorflow/tensorflow/cc/saved_model/loader.cc:240] Loading SavedModel with tags: { serve }; from: /home/work/ics-contrib/freechat-serving/serving/load/aS_24443.1000/2
2018-06-07 07:49:54.483091: I external/org_tensorflow/tensorflow/cc/saved_model/loader.cc:289] SavedModel load for tags { serve }; Status: fail. Took 199577 microseconds.
2018-06-07 07:49:54.483500: E tensorflow_serving/util/retrier.cc:38] Loading servable: {name: aS_24443.1000 version: 2} failed: Not found: Op type not registered 'ClipByValue' in binary running on sandbox03. Make sure the Op and Kernel are registered in the binary running in this process.

centOs
tensorflow1.8
keras 2.1

but i can start the serving on another computer which has the same environment

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
